programming-extensions/programming-extension-dataspaces/src/test/java/dataspaces/AbstractLimitingFileObjectTest.java

```diff
@@ -207,6 +207,8 @@ public void testReadOnlyMoveToFileObjectSource() throws FileSystemException {
         assertTrue(readOnlyFile.exists());
     }
 
+    // limitation of AbstractLimitingFileObject (rely on rename operation internally)
+    @Ignore
```

Comment:
It is a hint to make clear that it is no necessary or make no sense to test such a feature.